src/SamplesApp/SamplesApp.UITests/TestFramework/ImageAssert.cs

```diff
@@ -212,6 +217,45 @@ public static void HasColorAt(FileInfo screenshot, float x, float y, Color expec
 			}
 		}
 
+		public static void HasPixels(FileInfo screenshot, params ExpectedPixels[] expectations)
+		{
+			using (var bitmap = new Bitmap(screenshot.FullName))
+			{
+				foreach (var expectation in expectations)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'expectations' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4919453)